platforms/ios/example/Wysiwyg/Extensions/UIAlertController.swift

```diff
@@ -0,0 +1,32 @@
+// 
+// Copyright 2022 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+
+extension UIAlertController {
+    convenience init(alert: AlertConfig) {
+        self.init(title: alert.title, message: nil, preferredStyle: .alert)
+        addTextField()
+        //addTextField { $0.placeholder = alert.placeholder }
```

Comment:
Should we restore it or remove it?

Reply:
Yeah we definitely don't use this, thanks, I removed it.